src/Evolution/Executables/GeneralizedHarmonic/EvolveGhBinaryBlackHole.hpp

```diff
@@ -494,11 +497,19 @@ struct EvolutionMetavars {
                                                 use_control_systems>,
           Initialization::TimeStepperHistory<EvolutionMetavars>>,
       Initialization::Actions::NonconservativeSystem<system>,
-      Initialization::Actions::AddComputeTags<::Tags::DerivCompute<
-          typename system::variables_tag,
-          ::domain::Tags::InverseJacobian<volume_dim, Frame::ElementLogical,
-                                          Frame::Inertial>,
-          typename system::gradient_variables>>,
+      Initialization::Actions::AddComputeTags<
+          tmpl::list<::Tags::DerivCompute<typename system::variables_tag,
+                                          ::domain::Tags::InverseJacobian<
+                                              volume_dim, Frame::ElementLogical,
+                                              Frame::Inertial>,
+                                          typename system::gradient_variables>,
+                     // For observation horizon finds
+                     ::Tags::TimeAndPreviousCompute<
+                         static_cast<size_t>(::domain::ObjectLabel::A) + 10>,
```

Comment:
`typename AhA::temporal_id`, etc., to avoid the magic numbers.

Reply:
To do this I had to expose the index in the regular `TimeAndPrevious` tag and use that.

Reply:
Ah, I missed that this one was the compute tag and the other not.  Good solution, though.